### PR TITLE
Created density schema

### LINKED ISSUE #342: Create density schema
Create a density schema to describe densities of neuroscience objects such as neurons, cells, boutons in a given brain location.

### PR DESCRIPTION
**Pull request by annakristinkaufmann**
Fixes #342

### CODE REVIEW

**Comment by annakristinkaufmann** [comment]
@MFSY:  We could also model it as a density shape which has a brainLocation and a density property. That would allow us to validate data which contains a list of density values with the respective brainLocation information

File: shapes/neurosciencegraph/datashapes/core/density/schema.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+              "name": "Morphology type",
+              "description": "The morphology type of the neurons for which the density was measured.",
+              "node": "https://neuroshapes.org/commons/typedlabeledontologyterm/shapes/CellTypeOntologyTermShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:density",
+              "name": "Density",
+              "description": "Density of a neuroscience object such as cells, neurons or synaptic boutons.",
+              "node": "https://neuroshapes.org/commons/quantitativevalue/shapes/QuantitativeValueShape",
```
Comment (MFSY) [inline shapes/neurosciencegraph/datashapes/core/density/schema.json]: I recommend to have a local quantitativevalue shape and add a statistic property.

Comment (annakristinkaufmann) [reply shapes/neurosciencegraph/datashapes/core/density/schema.json]: Forgot to include that.

Comment (annakristinkaufmann) [reply shapes/neurosciencegraph/datashapes/core/density/schema.json]: Done.

**Pull request merged**